cmd/config/aws/create_role_cmd.go

```diff
@@ -11,14 +11,28 @@ import (
 	"github.com/spf13/cobra"
 	log "go.uber.org/zap"
 	"io"
+	"time"
+)
+
+var (
+	useDevTemplate bool
 )
 
 const (
-	configCreateRoleShort = "AWS Create Role via CloudFormation Stack"
-	//TODO For usage documentation, visit...
+	configCreateRoleShort   = "AWS Create Role via CloudFormation Stack"
 	configCreateRoleLong    = "Use AWS CloudFormation quick create to run a template that allows Armory to assume a role to manage deployments\n"
 	configCreateRoleExample = "armory config aws create-role"
-	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://armory-cdaas-cloudformation.s3.us-west-2.amazonaws.com/iam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack&param_AccountId=%s&param_ExternalId=%s"
+	templateUrl             = "https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1#/stacks/quickcreate?templateURL=https://s3.us-west-2.amazonaws.com/static.cloud.armory.io/templates/aws/cloudformation/%siam-role-cfn.template&stackName=Armory-CDAAS-Role-Stack-%s&param_AccountId=%s&param_ExternalId=%s&param_ArmoryRoleName=%s"
+	devTemplatePrefix       = "dev-"
+	armoryRoleName          = "ArmoryRole"
+
+	installPromptIntro  = "In order to deploy AWS resources, we need to create a Trust Relationship in your AWS account by adding a role that Armory can assume to execute deployments on your behalf."
+	installPromptPrereq = "Prerequisite: Log in to the AWS Management Console you want to connect to in your default browser. The logged in user requires access to configure IAM roles."
+	installPromptStep1  = "1. Type \"Y\" to begin the installation of resources. This will open your browser window with your AWS Console. You will complete the rest of this process in your browser."
```

Comment:
```suggestion
	installPromptStep1  = "1. Type \"Y\" to begin Stack creation. This opens your browser to the CloudFormation page of your AWS Console. You complete the rest of this process in your browser."
```

I'm trying to be a bit more explicit that "resources"